crates/core/src/parse.rs

```diff
@@ -480,8 +479,47 @@ pub async fn read_addon_directory<P: AsRef<Path>>(
         );
     }
 
+    // Addon folders that failed fingerprinting, but we can still build an addon
+    // using the wowi id from the `.toc`
+    let mut wowi_ids_from_nonmatch: Vec<_> = addon_folders
+        .iter()
+        .filter(|f| {
+            !fingerprint_addons.iter().any(|fa| {
+                fa.folders
+                    .iter()
+                    .any(|faf| faf.fingerprint == f.fingerprint)
+            })
+        })
+        .filter(|f| f.repository_identifiers.wowi.is_some())
```

Comment:
Do we need to make sure that tukui and curse id are None since we already created addons off any folders with those ids by this point?

Reply:
Good point, we properly do.

Reply:
Fixed now.

Reply:
Sweet! I still need to test but didn't notice anything else on reviewing the diff